src/location.c

```diff
@@ -524,16 +524,16 @@ handle_start_in_thread_func (GTask *task,
 
       g_variant_builder_init (&access_opt_builder, G_VARIANT_TYPE_VARDICT);
       g_variant_builder_add (&access_opt_builder, "{sv}",
-                             "deny_label", g_variant_new_string (_("Deny Access")));
+                             "deny_label", g_variant_new_string (_("Deny")));
       g_variant_builder_add (&access_opt_builder, "{sv}",
-                             "grant_label", g_variant_new_string (_("Grant Access")));
+                             "grant_label", g_variant_new_string (_("Allow")));
       g_variant_builder_add (&access_opt_builder, "{sv}",
                              "icon", g_variant_new_string ("find-location-symbolic"));
 
       if (g_str_equal (app_id, ""))
         {
-          title = g_strdup (_("Grant Access to Your Location?"));
-          subtitle = g_strdup (_("An application wants to use your location."));
+          title = g_strdup (_("Allow Access to Your Location?"));
+          subtitle = g_strdup (_("An application wants to know the location of this device."));
```

Comment:
Why the change? "your location" is usually fine, and it's only in very rare cases where *this device* needs to be specified.